lib/tasks/bundlers/generateLibraryPreload.js

```diff
@@ -48,15 +49,18 @@ function getBundleDefinition(namespace) {
 		};
 	}
 	return {
-		name: `${namespace}/library-preload.js`,
-		defaultFileTypes: [".js", ".fragment.xml", ".view.xml", ".properties", ".json"],
+		name: `${namespace}/library-preload${isDesignTime ? ".designtime" : ""}.js`,
```

Comment:
can we please address the handling of designtime bundles in a separate change. It totally makes sense, but I would suggest to follow the principle "1 topic - 1 change".

Reply:
sure I'll put that on the side